x-pack/test/security_api_integration/anonymous.config.ts

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
__note:__ didn't manage to find a way to write integration tests for the ApiKey credentials since we cannot know the key before FTR is started